### DISCUSSION TITLE
How to view details of shrinking and expanding gene families in a node (which gene families have changed) ？

### DISCUSSION THREAD

**Discussion by Jiahe-Sun** [hahnlab/CAFE5#89] [Q&A] [answered]
Hello,  
After reading the tutorial of cafe5, I have two doubts:
1: I am going to perform GO annotation on the shrinking and expanding gene family in a node? However, I can only see the number of contraction and expansion of a certain node family at present, and I cannot see which families are contracted and expanded. Can this information be obtained:

2: What do the tags such as <15>, <11>, <9>, <8> in Base_clade_results.txt in the result file mean? and how to map the nodes in the Base_clade_results.txt file to the nodes of the tree ?

Looking forward to your reply! Thanks in advance
[Base_clade_results.txt](https://github.com/hahnlab/CAFE5/files/9925503/Base_clade_results.txt)

**Comment by benfulton** [comment, marked as answer]
The Base_change.tab file is a tab-delimited file that shows the positive and negative changes for each family at each node.

The tags are internal identifiers assigned to each node by CAFE. The base_asr.tre file has Newick trees that show the location of each node with its internal identifier.

**Reply by Jiahe-Sun** [reply]
thank you!
